src/twisted/test/test_process.py

```diff
@@ -2340,6 +2340,64 @@ def test_findShebang(self):
         self.assertEqual(program, "/usr/bin/python")
 
 
+
+class Win32CreateProcessFlagsTests(unittest.TestCase):
+    """
+    Check the flags passed to CreateProcess.
+    """
+
+    @defer.inlineCallbacks
+    def test_flags(self):
+        """
+        Verify that the flags passed to win32process.CreateProcess() prevent a
+        new console window from being created. See http://tm.tl/5726
+        for a script to test this interactively.
+        """
+        from twisted.internet import _dumbwin32proc
+        flags = []
+        realCreateProcess = _dumbwin32proc.win32process.CreateProcess
+
+        def fakeCreateprocess(appName, commandLine, processAttributes,
+                              threadAttributes, bInheritHandles, creationFlags,
+                              newEnvironment, currentDirectory, startupinfo):
+            """
+            See the Windows API documentation for I{CreateProcess} for further details.
```

Comment:
twistedchecker shouldn't be checking a callback like this, and this documentation is all duplicative with the existing docs, so this docstring should probably be elided.